### PR TITLE
chore(documentation): add monitoring resources host details to API Doc

### PR DESCRIPTION
**Pull request by jeremyjaouen**
## Description
Add missing documentation and fix some errors.

**Fixes** # MON-5789

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 19.10.x
- [ ] 20.04.x
- [ ] 20.10.x
- [x] 21.04.x (master)

## Checklist

- [x] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: doc/API/centreon-api-v2.yaml [hunk 1]
```diff
@@ -524,6 +524,25 @@ paths:
           $ref: '#/components/responses/Forbidden'
         '500':
           $ref: '#/components/responses/InternalServerError'
+  /monitoring/resources/hosts/{host_id}:
+    get:
+      tags:
+        - Resource
+      summary: "Get a host resources informations"
```
Comment (kduret) [inline doc/API/centreon-api-v2.yaml:531]:
```suggestion
      summary: "Get a host resources information"
```

**Review by kduret — DISMISSED** [review]

File: doc/API/centreon-api-v2.yaml [hunk 2]
```diff
@@ -3510,6 +3529,82 @@ components:
...
+          format: integer
+          description: "Number of notifications"
+          example: 3
+        next_check:
+          type: string
+          nullable: true
+          format: date-time
+          description: "next planned check"
+          example: "2020-11-23T10:26:19+01:00"
+        perofrmance_data:
```
Comment (sc979) [inline doc/API/centreon-api-v2.yaml:3588]:
```suggestion
        performance_data:
```

File: doc/API/centreon-api-v2.yaml [hunk 3]
```diff
@@ -3510,6 +3529,82 @@ components:
...
+          type: string
+          description: "Details of performances"
```
Comment (sc979) [inline doc/API/centreon-api-v2.yaml:3590]:
```suggestion
          description: "Details of performance"
```

File: doc/API/centreon-api-v2.yaml [hunk 4]
```diff
@@ -3510,6 +3529,82 @@ components:
...
+          example: "rta=0,098ms;3000,000;5000,000;0; pl=0%;80;100;0;100 rtmax=0,098ms;;;; rtmin=0,098ms;;;;"
+        execution_time:
+          type: number
+          format: float
+          description: "duration of check"
+          example: 0.005618
+        latency:
+          type: number
+          format: float
+          description: "Latency of the host"
```
Comment (sc979) [inline doc/API/centreon-api-v2.yaml:3600]:
```suggestion
          description: "Host Latency"
```

File: doc/API/centreon-api-v2.yaml [hunk 5]
```diff
@@ -524,6 +524,25 @@ paths:
           $ref: '#/components/responses/Forbidden'
         '500':
           $ref: '#/components/responses/InternalServerError'
+  /monitoring/resources/hosts/{host_id}:
+    get:
+      tags:
+        - Resource
+      summary: "Get a host resources information"
```
Comment (sc979) [inline doc/API/centreon-api-v2.yaml:531]:
```suggestion
      summary: "Get information on host resources"
```

File: doc/API/centreon-api-v2.yaml [hunk 6]
```diff
@@ -3510,6 +3529,82 @@ components:
...
+      properties:
+        command_line:
+          type: string
+          nullable: true
+          description: "Check command of the host"
+          example: "/usr/lib64/nagios/plugins/check_icmp -H 127.0.0.1 -w 3000.0,80% -c 5000.0,100% -p 1"
+        poller_name:
+          type: string
+          nullable: true
+          description: "Name of the monitoring server of the Host"
```
Comment (sc979) [inline doc/API/centreon-api-v2.yaml:3545]:
```suggestion
          description: "Name of the monitoring server of the host"
```

File: doc/API/centreon-api-v2.yaml [hunk 7]
```diff
@@ -3510,6 +3529,82 @@ components:
...
+          example: "Central"
+        timezone:
+          type: string
+          description: "Timezone used by Host"
```
Comment (sc979) [inline doc/API/centreon-api-v2.yaml:3549]:
```suggestion
          description: "Timezone used by the host"
```

File: doc/API/centreon-api-v2.yaml [hunk 8]
```diff
@@ -3510,6 +3529,82 @@ components:
...
+          example: "Europe/Paris"
+        flapping:
+          type: boolean
+          description: "Host is flapping or not"
+          example: false
+        percent_state_change:
+          type: number
+          format: float
+          description: "Percent of Host state changing"
```
Comment (sc979) [inline doc/API/centreon-api-v2.yaml:3558]:
```suggestion
          description: "Percent of Host state change"
```

Comment (sc979) [reply doc/API/centreon-api-v2.yaml:3558]: Not sure about this one

File: doc/API/centreon-api-v2.yaml [hunk 9]
```diff
@@ -3510,6 +3529,82 @@ components:
...
+          type: number
+          nullable: true
+          format: integer
+          description: "Number of notifications"
+          example: 3
+        next_check:
+          type: string
+          nullable: true
+          format: date-time
+          description: "next planned check"
```
Comment (sc979) [inline doc/API/centreon-api-v2.yaml:3586]:
```suggestion
          description: "Next planned check"
```

File: doc/API/centreon-api-v2.yaml [hunk 10]
```diff
@@ -3510,6 +3529,82 @@ components:
...
+          example: 1
+        chart_url:
+          type: string
+          nullable: true
+          description: "Url of chart"
+        last_notification:
+          type: string
+          nullable: true
+          format: date-time
+          description: "DateTime of the last notification"
```
Comment (sc979) [inline doc/API/centreon-api-v2.yaml:3574]:
```suggestion
          description: "Last notification date and time in ISO format"
```

File: doc/API/centreon-api-v2.yaml [hunk 11]
```diff
@@ -3510,6 +3529,82 @@ components:
...
+          description: "next planned check"
+          example: "2020-11-23T10:26:19+01:00"
+        perofrmance_data:
+          type: string
+          description: "Details of performances"
+          example: "rta=0,098ms;3000,000;5000,000;0; pl=0%;80;100;0;100 rtmax=0,098ms;;;; rtmin=0,098ms;;;;"
+        execution_time:
+          type: number
+          format: float
+          description: "duration of check"
```
Comment (sc979) [inline doc/API/centreon-api-v2.yaml:3595]:
```suggestion
          description: "Check duration"
```

File: doc/API/centreon-api-v2.yaml [hunk 12]
```diff
@@ -3510,6 +3529,82 @@ components:
...
+        criticality:
+          type: number
+          nullable: true
+          format: integer
+          description: "Criticality level"
+          example: 1
+        chart_url:
+          type: string
+          nullable: true
+          description: "Url of chart"
```
Comment (sc979) [inline doc/API/centreon-api-v2.yaml:3569]:
```suggestion
          description: "Chart URL"
```

File: doc/API/centreon-api-v2.yaml [hunk 13]
```diff
@@ -3510,6 +3529,82 @@ components:
...
+          nullable: true
+          description: "Name of the monitoring server of the Host"
+          example: "Central"
+        timezone:
+          type: string
+          description: "Timezone used by Host"
+          example: "Europe/Paris"
+        flapping:
+          type: boolean
+          description: "Host is flapping or not"
```
Comment (sc979) [inline doc/API/centreon-api-v2.yaml:3553]:
```suggestion
          description: "Is host flapping"
```

**Review by sc979 — APPROVED** [review]

**Review by kduret — APPROVED** [review]

**Pull request merged**